ChaosMod/Components/Globals.cpp

```diff
@@ -0,0 +1,71 @@
+#include <stdafx.h>
+
+#include "Globals.h"
+
+#include "Lib/scrThread.h"
+
+#include "Memory/Script.h"
+
+std::map<std::string, int> Globals::m_GlobalsIndexMap = {};
+static std::vector<GlobalRegistration> m_GlobalsRegistration = {};
+
+Globals::Globals() : Component()
+{
+	REGISTER_GLOBAL("CellPhoneState", "dialogue_handler",
+	                "2D 00 02 00 ? 52 ? ? 41 ? 72 08 2A 06 56 ? ? 52 ? ? 41 ? 71 08 20 56 ? ? 72", 6,
+	                GlobalPatternIdiom::GLOBAL_U16);	
+	
+	REGISTER_GLOBAL("CellPhoneLock", "dialogue_handler",
+	                "72 54 ? ? 5d ? ? ? 71 2c ? ? ? 2b 72 54 ? ? 77 54 ? ? 52 ? ? 25 ? 2c ? ? ? 53 ? ? 06 56 ? ? 52 ? ? 76", 2,
+	                GlobalPatternIdiom::GLOBAL_U16);
```

Comment:
Might be better to search for those in the effect directly I think? Otherwise this is just going to clutter up this file if we have too many of those. Also it's more consistent to the lazy search approach we have with patterns in the codebase.